core/systems/advert/submission_view.ex

```diff
@@ -257,13 +219,29 @@ defmodule Systems.Advert.SubmissionView do
             <.spacing value="M" />
 
             <div class="flex flex-col gap-14">
-              <%= for field <- Map.keys(@inclusion_labels) do %>
+            <%!-- This is responsible for rendering all selector childs, with their respective options. --%>
+              <%= for field <- Map.keys(@selector_option_labels) do %>
+                <div>
+                  <Text.title4><%= inclusion_criterium_title(field) %></Text.title4>
+                  <.spacing value="S" />
+                  <.child name={field} fabric={@fabric} />
+                </div>
+              <% end %>
+
               <div>
-                <Text.title4><%= inclusion_title(field) %></Text.title4>
+              <%!-- This renders the min-max birth years number inputs --%>
```

Comment:
Comments in UI could be indicating a function component with a self explanatory name would be a better solution